File: src/URI/URI_1008.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1008 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n, horas;
		double valorhra, salary;
		
		n = sc.nextInt();
		horas = sc.nextInt();
		valorhra = sc.nextDouble();
		salary = valorhra * horas;
		
		System.out.println("NUMBER = " + n);
		System.out.printf("SALARY = U$ %.2f%n", salary);
		
		
		
		
		sc.close();
	}

}

File: kotlin/URI_1008.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val number = sc.nextInt()
    val hours = sc.nextInt()
    val valueHour = sc.nextDouble()
    println("NUMBER = $number\nSALARY = U$ ${"%.2f".format(hours * valueHour)}")
}